docs/guides/skins.md

```diff
@@ -3,22 +3,23 @@ Skins
 
 ## Base Skin
 The base Video.js skin is made using HTML and CSS (although we use the [Sass preprocessor](http://sass-lang.com)),
-and by default these styles are added to the dom for you!
+and by default these styles are added to the DOM for you!
 That means you can build a custom skin by simply taking advantage of the cascading aspect of CSS and overriding
 the styles you'd like to change.
 
 If you don't want Video.js to inject the base styles for you, you can disable it by setting `window.VIDEOJS_NO_BASE_THEME = true` before Video.js is loaded.
 Keep in mind that without these base styles enabled, you'll need to manually include them.
+
 Video.js does not currently include the base skin automatically yet, so, this option isn't necessary.
 
 ## Default style elements
-Video.js uses a couple of dynamically, specifically, there's a default styles element as well as a player dimensions style element.
+Video.js uses a couple of style elements dynamically, specifically, there's a default styles element as well as a player dimensions style element.
```

Comment:
Heh, I think both are technically correct though a bit ambiguous. I was going for ((default styles) element) as opposed to a (default (style element)).
Maybe I should just change it to `default styles style element`?